styles/app/index.styl

```diff
@@ -77,6 +77,16 @@ body {
   bottom: 0
   left: brandWidth
 
+.navbar-search-container
+  position: absolute;
+  top: 0;
+  right: 0;
+  max-width: 400px;
+  height: 100%;
+  line-height: 1em;
```

Comment:
This line-height is the one change since I showed the search box to you earlier today.

The Derby site in general uses a larger line-height, which caused the search box to be too tall. This resets the line-height to normal for the search box.